docs/06_Oversikt_over_klasser.adoc

```diff
@@ -99,44 +126,57 @@
 [cols="30s,70d"]
 |===
 | Engelsk navn | License document
-| Beskrivelse | En juridisk dokument som gir offisiell tillatelse til å gjøre noe med en ressurs.
+| Beskrivelse | Et juridisk dokument som gir offisiell tillatelse til å gjøre noe med en ressurs.
 | URI | dct:LicenseDocument
 | Referanse | http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=terms#LicenseDocument
 | Status | Anbefalt
 |===
 
-== Valgfrie klasser
+=== Regel [[klasse-regel]]
 
-=== Katalogpost [[klasse-katalogpost]]
+[cols="30s,70d"]
+|===
+| Engelsk navn | Rule
+| Beskrivelse | Brukes til å representere regler/dokumenter som setter spesifikke retningslinjer eller prosedyrer som skal følges av et offentlig basisregister. Det kan inkludere krav til informasjon som forvaltes og tjenester som tilbys av et offentlig basisregister.
```

Comment:
Litt usikker på hva et basisregister er? Er det grunnregister? Og gjelder Rule kun for denne type register?

Reply:
Her har BRegDCAT-AP beskrevet for snevert. Skal rette opp.